### PR TITLE
Update cops_metrics.adoc

### PR DESCRIPTION
**Pull request by alexmcdermid**
Small inconsistencies in wording and grammar between metric cop descriptions.

-----------------

Before submitting the PR make sure the following are checked:

* [x] The PR relates to *only* one subject with a clear title and description in grammatically correct, complete sentences.
* [x] Wrote [good commit messages][1].
* [x] Commit message starts with `[Fix #issue-number]` (if the related issue exists).
* [x] Feature branch is up-to-date with `master` (if not - rebase it).
* [x] Squashed related commits together.
* [ ] Added tests.
* [x] Ran `bundle exec rake default`. It executes all tests and runs RuboCop on its own code.
* [ ] Added an entry (file) to the [changelog folder](https://github.com/rubocop/rubocop/blob/master/changelog/) named `{change_type}_{change_description}.md` if the new code introduces user-observable changes. See [changelog entry format](https://github.com/rubocop/rubocop/blob/master/CONTRIBUTING.md#changelog-entry-format) for details.

[1]: https://chris.beams.io/posts/git-commit/

### CODE REVIEW

File: docs/modules/ROOT/pages/cops_metrics.adoc [hunk 1]
```diff
@@ -446,7 +446,7 @@ end               # 6 points
 | 0.87
 |===
 
-Checks if the length a module exceeds some maximum value.
+Checks if the length of a module exceeds some maximum value.
```
Comment (koic) [inline docs/modules/ROOT/pages/cops_metrics.adoc:449]: This docs/modules/ROOT/pages/cops_metrics.adoc is auto-generated file. Please edit lib/rubocop/cop/metrics/class_length.rb original file instead. So, edits to this adoc file is not required.

Comment (alexmcdermid) [reply docs/modules/ROOT/pages/cops_metrics.adoc]: Thank you for the clarification, I've made the change.

**Comment by koic** [comment]
This looks good! Can you squash your commits into one?

**Comment by alexmcdermid** [comment]
> This looks good! Can you squash your commits into one?

Yes!

**Comment by koic** [comment]
Thanks!

**Pull request merged**